### PR TITLE
Add issues templates and some other github related files

### LINKED ISSUE #8: Add templates for GH Issues

### PR DESCRIPTION
**Pull request by davidrabko**
# Pull Request

## Description

Now we have a bug and issue request and

* SECURITY.md for security breaches reports info
* CONTRIBUTING.md for the future))
* CODE_OF_CONDUCT.md 

## Related Issue

#8 

## Checklist

Please review and check all applicable items:

- [x] I have tested my changes thoroughly
- [x] My code follows the project's coding standards and guidelines
- [x] I have reviewed my changes to ensure there are no unnecessary changes
- [x] My pull request has a descriptive title and includes a detailed description of the changes

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Working on Issues
+If you would like to work on an existing issue, please follow these steps:
+
+1. Assign the issue to yourself to indicate that you are working on it
+2. Write a comment on the issue to let others know that you are working on it
+3. Fork the repository and create a new branch for your changes
+4. Make your changes and create a pull request following the steps outlined above
+
+# Code Style
+Please ensure that your code adheres to the project's [coding standards and guidelines](https://github.com/RomanMager/ypm/blob/main/.editorconfig).
```
Comment (daverbk) [inline .github/CONTRIBUTING.md:60]: trust me the link will work when we merge it to main))

File: .github/ISSUE_TEMPLATE/1-bug.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/RomanMager/ypm/blob/main/.github/CODE_OF_CONDUCT.md)
```
Comment (davidrabko) [inline .github/ISSUE_TEMPLATE/1-bug.yml:60]: trust me the link will work when we merge it to main))

File: .github/ISSUE_TEMPLATE/2-feature-request.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        - Medium
+        - High
+      default: 1
+    validations:
+      required: true
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/RomanMager/ypm/blob/main/.github/CODE_OF_CONDUCT.md)
```
Comment (davidrabko) [inline .github/ISSUE_TEMPLATE/2-feature-request.yml:45]: this one too)

File: .github/SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, please e-mail davidryabko13@gmail.com with a description of the issue, the steps you took to create the issue, affected versions, and if known, mitigations for the issue
```
Comment (davidrabko) [inline .github/SECURITY.md:5]: we can add or change my email to urs if ud like

Comment (leingenm) [reply .github/SECURITY.md:5]: We can leave it as it is

**Review by leingenm — APPROVED** [review]

**Pull request merged**